recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -11,8 +11,5 @@ conda config --set add_pip_as_python_dependency false
 conda update -n root --yes --quiet conda
 conda install -n root --yes --quiet jinja2 conda-build anaconda-client
 
-:: KLUDGE to work around changes in conda-build 2.0.0
-conda install -n root --yes --quiet conda-build=1
```

Comment:
So I think we still need these kludgy pinnings for now as there isn't a way to re-render with the upload script. Going to add them back in.